prowler/providers/aws/services/apigateway/apigateway_service.py

```diff
@@ -107,6 +151,11 @@ class Stage(BaseModel):
     tags: Optional[list] = []
 
 
+class PathResourceMethods(BaseModel):
+    path: str
+    resource_methods: dict
```

Comment:
Can we create a type for this attribute?

Reply:
In this case since we are only getting that info adding a type for that info complicates more the check's logic.
In the future if we need to retrieve more info from the resources we will include the model